[No QA] Update help docs for Release 1: Automatic report reassignment on workflow changes (#97799)
Expensify/App: docs/articles/new-expensify/workspaces/Add-Approvals.md

3. Select a new member or deselect one to remove them.
4. Click **Save**.

> **Note:** When you change an approver in a workflow, any reports currently pending approval with the previous approver are automatically reassigned to the new approver. A system message will appear in the report confirming the reassignment. Reports are also reassigned when you remove an approver from the workspace or shorten an approval chain. Some reports are intentionally excluded — see the FAQ below for the full list of triggers and exclusions.

Reviewer: ```suggestion
**Note:** When you change an approver in a workflow, any Outstanding reports assigned to the approver you replaced are reassigned to the new approver. A system message appears in the report confirming the reassignment. Reports are also reassigned when you remove an approver from the workspace or shorten an approval chain. Some reports are intentionally excluded — see the FAQ below for the full list of triggers and exclusions.
```

## What happens to pending reports when I change an approver?

> Any reports currently awaiting approval from the previous approver are automatically reassigned to the correct new approver, with no prompt or confirmation required. A system message appears in each report indicating the approver was reassigned. This applies to reports that are already in the approval process, not just new reports, and it happens in these situations:

Reviewer: ```suggestion
Outstanding reports are automatically reassigned to the new approver, with no prompt or confirmation required. A system message appears in each report indicating the approver was reassigned. This applies to reports that are already in the approval process, not just new reports, and it happens in these situations:
```

- **You change an approver in a workflow:** reports pending with the previous approver move to the new approver.
- **You remove an approver from the workspace:** if you remove a member who is an approver on pending reports, those reports are reassigned to the **workspace owner**.
- **You shorten an approval chain:** if you remove a downstream approver from a chain, reports already partway through move to the appropriate remaining approver. For example, in an Alice → Bob → Claire chain where Bob has already approved and you remove Claire, the report returns to Bob for final approval.
- **The workflow changes through an integration:** the same reassignment happens when the workflow changes through an HR integration such as Gusto or TriNet, or through a bulk member update. In that case, the report's system message indicates the change came from the integration rather than from a specific person.

> ## Are there cases where reports are not reassigned?

Reviewer: N/A, this is an FAQ so that rule doesn't apply

Yes. Reports are not automatically reassigned in these cases:

- **The current approver is a Category or Tag approver:** reports awaiting a Category or Tag approver are not reassigned. Those approval steps run before, and take precedence over, the workspace approval workflow.

> - **The workspace uses Advanced Approval:** automatic reassignment does not run for workspaces using Advanced Approval.

Reviewer: " automatic reassignment does not run for workspaces using Advanced Approval." 

Is this true, @garrettmknight? One of the above examples references an advanced approval chain, so we should probably update either the example, or the wording here.

> - **The workspace uses Advanced Approval:** automatic reassignment does not run for workspaces using Advanced Approval.

Reviewer 2: It's more nuanced than that. It works for most Advanced Approval changes. I think I'll add more detail here.
